python/lsst/analysis/tools/actions/plot/plotUtils.py

```diff
@@ -598,6 +598,68 @@ def addSummaryPlot(fig, loc, sumStats, label):
     return fig
 
 
+def shorten_list(numbers: Iterable[int]) -> str:
+    """Shorten a list of integers.
+
+    Parameters
+    ----------
+    numbers : `list` [`int`]
```

Comment:
Rather than ``list``, should this just be `iterable` (without back ticks)? From the doc string and the unit test, it looks like you're happy accepting any iterable?